app/controllers/concerns/verify_profile_concern.rb

```diff
@@ -0,0 +1,31 @@
+module VerifyProfileConcern
+  private
+
+  def profile_or_verify_profile_path
+    public_send "#{profile_or_verify_profile_route}_path"
+  end
+
+  def profile_or_verify_profile_url
+    public_send "#{profile_or_verify_profile_route}_url"
+  end
+
+  def profile_or_verify_profile_route
```

Comment:
now that we've made the name `account` for the default page, should we call it `account_or_verify_profile_route`?